root.tf

```diff
@@ -733,3 +799,34 @@ module "create_bulk_users_bucket" {
   lambda_notification = true
   lambda_arn          = module.create_keycloak_users_s3_lambda.create_keycloak_users_s3_lambda_arn
 }
+
+module "keycloak_user_management_github_environment" {
+  source          = "./tdr-terraform-modules/github_environments"
+  environment     = local.environment
+  repository_name = "nationalarchives/tdr-keycloak-user-management"
+  team_slug       = "transfer-digital-records-admins"
+  secrets = {
+    TITLE_STAGE            = title(local.environment)
+    ACCOUNT_NUMBER         = data.aws_caller_identity.current.account_id
+    MANAGEMENT_ACCOUNT     = data.aws_ssm_parameter.mgmt_account_number.value
+    SLACK_FAILURE_WORKFLOW = data.aws_ssm_parameter.slack_failure_workflow.value
+    SLACK_SUCCESS_WORKFLOW = data.aws_ssm_parameter.slack_success_workflow.value
+    WORKFLOW_PAT           = data.aws_ssm_parameter.workflow_pat.value
+  }
+}
+
+module "deploy_lambda_github_actions_policy" {
```

Comment:
Maybe rename slightly to provide a consistent naming convention for the github stuff: `github_actions_deploy_lambda_policy` ?